tests/test_2d_gradient/results_true.dat

```diff
@@ -1,203 +1,203 @@
 # Iterations: 57
 keff:  3.39805E-01
 fluxes:
-2.244961E+00
-8.949252E-01
-2.197271E+00
-8.789067E-01
-2.197271E+00
-8.789067E-01
-2.105939E+00
+2.244960E+00
+8.949250E-01
+2.197270E+00
+8.789065E-01
+2.197270E+00
+8.789065E-01
+2.105938E+00
+8.461863E-01
+2.152172E+00
+8.621911E-01
+2.105938E+00
 8.461863E-01
-2.152173E+00
-8.621913E-01
-2.105939E+00
-8.461864E-01
 1.979218E+00
 7.983337E-01
 2.069855E+00
-8.315059E-01
+8.315058E-01
 2.069855E+00
-8.315060E-01
+8.315058E-01
 1.979218E+00
-7.983339E-01
+7.983337E-01
 1.830133E+00
 7.384444E-01
-1.958458E+00
-7.875505E-01
-2.003656E+00
-8.037618E-01
-1.958458E+00
-7.875505E-01
+1.958457E+00
```

Comment:
They were recomputed with:

    python update_results.py -R test_2d_gradient

so the code was rebuilt and the test was run. It looks like `update_results.py` uses double precision.